src/modules/remove.js

```diff
@@ -0,0 +1,4 @@
+export default function removeTask(arr, index) {
+  arr.splice(index, 1);
+  return arr;
+}
```

Comment:
Here also it would be good if you use an arrow function instead of a regular one, since the purpose of an arrow function provides a concise way to write functions in JavaScript.